pysatNASA/tests/test_instruments.py

```diff
@@ -96,6 +96,23 @@ def test_load_cdflib(self, inst_dict):
 
         return
 
+    @pytest.mark.second
+    @pytest.mark.parametrize("inst_dict", instruments['cdf'])
+    @pytest.mark.skipif(cdflib_only,
+                        reason=" ".join(("Additional load tests not required",
+                                         "when pysatCDF not installed")))
```

Comment:
```suggestion
```
I think we should test this either way.  Note that Github Actions currently does not install pysatCDF, so this would never be run